src/Features/Core/Portable/CodeRefactorings/InvertIf/AbstractInvertIfCodeRefactoringProvider.cs

```diff
@@ -81,264 +64,6 @@ public override async Task ComputeRefactoringsAsync(CodeRefactoringContext conte
                     document, model, ifStatement, cancellationToken));
         }
 
-        internal SyntaxNode Negate(
-            SyntaxNode expression,
-            SyntaxGenerator generator,
-            ISyntaxFactsService syntaxFacts,
-            SemanticModel semanticModel,
-            CancellationToken cancellationToken)
```

Comment:
All this code moved, virtually unchanged from the invert-if feature, into a helper extension method.  that way it can be used by this feature, or hte 'use conditional expression' feature.